language-support/ts/daml-ledger/package.json

```diff
@@ -23,12 +23,13 @@
     "cross-fetch": "^3.0.4",
     "events": "^3.1.0",
     "isomorphic-ws": "^4.0.1",
+    "prettier": "^2.0.2",
```

Comment:
Do we really need this? If we do, it should be a `devDependency`.

Reply:
no we don't, this seems to have slipped in. removed it.